### ISSUE TITLE
Use Html or Template on legend and label

### ISSUE THREAD

**Issue by iceoz** [swimlane/ngx-charts#627]
i'm trying to use raw html on legend and in the label, but when i pass a img tag or a html especial carcter the componente scapes, even if i use the the label template.
```
[ ] bug report => search github for a similar issue or PR before submitting
[x] feature request
[ ] support request => Please do not submit support request here
```

**Current behavior**
Pass a html tag to legend or label and the tag appear like a string not like a a element

**Expected behavior**
Pass a img tag to legend and appear a image on legend

**Comment by georgewilde** [comment]
I agree it would be good to be able to return HTML from the labelFormatting function and this is rendered as HTML and not as a string.
